pkgs/development/python-modules/nanobind/default.nix

```diff
@@ -18,25 +18,26 @@
 }:
 buildPythonPackage rec {
   pname = "nanobind";
-  version = "1.9.2";
+  version = "2.0.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "wjakob";
-    repo = pname;
-    rev = "v${version}";
-    hash = "sha256-6swDqw7sEYOawQbNWD8VfSQoi+9wjhOhOOwPPkahDas=";
+    repo = "nanobind";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-yDFrhSIWywWw7ri5aHRPigi9ujDeazpJa4AVrkLx5BI=";
     fetchSubmodules = true;
   };
 
   disabled = pythonOlder "3.8";
 
-  nativeBuildInputs = [
+  build-system = [
```

Comment:
Thanks a lot for your cleanup and update! Could quickly explain to me why you did this change and what it implies? Many thanks :)

Reply:
The naming in the "pythonPackages space" was aligned with PEP-517. `build-system` should now contains the actual build system, e. g., `setuptools`, `hatchling`, `poetry-core`, etc.. `nativeBuildInputs`still works, same goes for `propagatedBuildInputs` which was replaced by `dependencies`.